.github/workflows/documentation-checks.yaml

```diff
@@ -60,3 +60,10 @@ jobs:
         uses: canonical/documentation-workflows/linkcheck@main
         with:
           working-directory: ${{ inputs.working-directory }}
+
+      - name: Accessibility Check
+        id: pa11y-step
+        if: success() || failure()
+        uses: canonical/documentation-workflows/pa11y@main
+        with:
+          working-directory: ${{ inputs.working-directory }}
```

Comment:
Is there a way to make this check NOT fail the build?
If so, we should do this at least for now. Otherwise, everyone who is using the action (which hopefully is everyone who uses the starter pack) will get failures until they update their doc sets to include the Makefile target and the fixes, and potentially until they fix any additional issues they see.
And while it's generally not nice to break things for everyone, we definitely shouldn't do it right before the Christmas break. ;)

Reply:
Eventually yes, of course. But the check needs to be stable (thus expected to run without failures also on bigger - non-customised - doc sets) before we can make it required.

Reply:
Well, we want everyone to run the check, and eventually we want it to pass for all the input. So I think moving it into a separate workflow isn't a good idea, because most teams won't switch.

Having it cause red checks is also a bad idea, because some teams have conventions in place to never merge anything with a red check - plus, even if a team doesn't have that convention, who would ever click on the result to confirm that it's still the accessibility check that is red and not something else this time?

So I think the best solution (for now!) is to include it but make sure that no matter if it passes or fails, it won't make the workflow run fail. That's [continue-on-error](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstepscontinue-on-error) if I remember correctly.
That way, teams will have the output available to fix violations (and if we have the time, we can check for doc sets that have violations, help fix them, and include the fixes in the starter pack where feasible), and once we feel that the check is stable for most inputs, we can make it required.